src/Form/countryNames.js

```diff
@@ -1,255 +1,562 @@
-// Country names object using 2-letter country codes to reference country name
-// ISO 3166 Alpha-2 Format: [2 letter Country Code]: [Country Name]
-// Sorted alphabetical by country name (special characters on bottom)
-// Source: https://gist.github.com/incredimike/1469814#file-variouscountrylistformats-js-L272
+/**
+ * Country names object using 2-letter country codes to reference country name
+ * Derived from the Intl.DisplayNames implementation
+ * @source https://stackoverflow.com/a/70517921/1948947
```

Comment:
I've switched to using the `Intl.DisplayNames` mainly because we use it for filling the data. If we didn't use the exact same set of names we might have ended up autofilling a field and the same field wouldn't be available for storage. It's an edge case, but this solves it.

For context, we only use this list when `Intl.DisplayNames` itself is not available in the browser.